Лабораторная работа 2/task3.5.2.py

```diff
@@ -0,0 +1,11 @@
+# TODO Напишите функцию find_common_participants
+list_=()
+def find_common_participants(participants_group):
+
+    print(participants_first_group.intersection(participants_second_group))
+
+    return find_common_participants(participants_first_group,participants_second_group)
+participants_first_group = "Иванов|Петров|Сидоров"
+participants_second_group = "Петров|Сидоров|Смирнов"
+
```

Comment:
Поскольку в объявленной ранее функции `def find_common_participants` переменная `delimiter =','` равна запятой, то при вызове функции необходимо явно указать новое значение параметра `delimiter`

```suggestion
participants = find_common_participants(participants_first_group, participants_second_group, delimiter = '|')
```

В конце останется только напечатать результат работы функции